LiteFarmOrg/LiteFarm
# [Lf 2458, LF-2459] : Feature - building bulk sensor upload modal

This PR is dependent on [LF-2460-bulk-create-sensors](https://github.com/LiteFarmOrg/LiteFarm/tree/LF-2460-bulk-create-sensors). Please review that one before starting with this.

Also, 
1. run pnpm i
2. review the files only related to webapp/frontend.

To Test:
1. It's good to check the details in the ticket. 
https://lite-farm.atlassian.net/browse/LF-2458
https://lite-farm.atlassian.net/browse/LF-2459
